pkg/scheduler/util/utils_test.go

```diff
@@ -207,3 +211,91 @@ func TestGetContainerPorts(t *testing.T) {
 		}
 	}
 }
+
+func TestGetPodFullName(t *testing.T) {
+	pod := &v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: "test",
+			Name:      "pod1",
+		},
+	}
+	got := GetPodFullName(pod)
+	expected := fmt.Sprintf("%s_%s", pod.Name, pod.Namespace)
+	if got != expected {
+		t.Errorf("Got wrong full name, got %s, expected: %s", got, expected)
+	}
+}
+
+func TestGetEarliestPodStartTime(t *testing.T) {
+	prio1, prio2 := int32(1), int32(2)
+
+	pod1 := &v1.Pod{
+		Spec: v1.PodSpec{
+			Priority: &prio1,
+		},
+		Status: v1.PodStatus{
+			StartTime: &metav1.Time{Time: time.Now()},
+		},
+	}
+	pod2 := &v1.Pod{
+		Spec: v1.PodSpec{
+			Priority: &prio2,
+		},
+		Status: v1.PodStatus{
+			StartTime: &metav1.Time{Time: time.Now().Add(time.Second)},
+		},
+	}
+	pod3 := &v1.Pod{
+		Spec: v1.PodSpec{
+			Priority: &prio2,
+		},
+		Status: v1.PodStatus{
+			StartTime: &metav1.Time{Time: time.Now()},
+		},
+	}
+	victims := &api.Victims{
+		Pods: []*v1.Pod{
+			pod1,
+			pod2,
+			pod3,
+		},
+		NumPDBViolations: 1,
+	}
+	startTime := GetEarliestPodStartTime(victims)
+	if !startTime.Equal(pod3.Status.StartTime) {
+		t.Errorf("Got wrong earliest pod start time")
+	}
+}
+
+func TestMoreImportantPod(t *testing.T) {
+	prio1, prio2 := int32(1), int32(2)
+
+	pod1 := &v1.Pod{
```

Comment:
We could create a function named `newPrioirtyPodWithStartTime(priority int32, time.Time) *v1.Pod` which could help to remove the duplicate pattern.

```go
pod1 := newPrioirtyPodWithStartTime(1, time.Now())
pod1 := newPrioirtyPodWithStartTime(2, time.Now().Add(time.Second))
```

Feel free to change the function name if you have a better one.

Reply:
I have moved the variables inside the test functions, and added a function `newPrioirtyPodWithStartTime(priority int32, time.Time) *v1.Pod` to build pod.